### PR TITLE
fix: Main transfer animation improvement

### PR DESCRIPTION
**Pull request by juan-cortes**
### 📝 Description

Improve the animation of the main transfer CTA on LLM

### ❓ Context

- **Impacted projects**: `ledger-live-mobile` 
- **Linked resource(s)**: `https://ledgerhq.atlassian.net/browse/LIVE-5827` 

### ✅ Checklist

- [ ] **Test coverage** 
- [ ] **Atomic delivery** 
- [x] **No breaking changes** 

### 📸 Demo

https://user-images.githubusercontent.com/4631227/220573242-d8b6e0bf-2a85-477f-845b-12cf4eb9b293.mov

https://user-images.githubusercontent.com/4631227/220573250-280a2228-e414-446e-aee6-65dd7e5e32a7.mov

### 🚀 Expectations to reach
Just look at the animations and make sure nothing else is broken

### CODE REVIEW

File: apps/ledger-live-mobile/src/components/TabBar/Transfer.tsx [hunk 1]
```diff
@@ -146,10 +146,13 @@ export function TransferTabIcon() {
...
 
   const closeModal = useCallback(() => {
-    setIsOpened(false);
+    const animCallback = () => {
+      setIsOpened(false);
+    };
     openAnimValue.value = withTiming(2, animParams, finished => {
       if (finished) {
         openAnimValue.value = 0;
+        runOnJS(animCallback)();
```
Comment (ofreyssinet-ledger) [inline apps/ledger-live-mobile/src/components/TabBar/Transfer.tsx:155]: michael-scott-the-office

Comment (juan-cortes) [reply apps/ledger-live-mobile/src/components/TabBar/Transfer.tsx:155]: I miss the office

**Review by ofreyssinet-ledger — APPROVED** [review]
great thanks for fixing that 🙏

File: apps/ledger-live-mobile/src/components/TabBar/Transfer.tsx [hunk 2]
```diff
@@ -76,7 +76,7 @@ const BackdropPressable = Animated.createAnimatedComponent(styled(Pressable)`
   background-color: rgba(0, 0, 0, 0.7);
 `);
 
-const DURATION_MS = 250;
+const DURATION_MS = 400;
```
Comment (Justkant) [inline apps/ledger-live-mobile/src/components/TabBar/Transfer.tsx:79]: I liked it a bit faster

**Review by Justkant — APPROVED** [review]
Thx for this fix, was it necessary to change the duration ?

**Comment by juan-cortes** [comment]
> Thx for this fix, was it necessary to change the duration ?

I synced with design on it, feel free to ask another designer and let them fight over it :trollface: 
I personally like it like this.

**Pull request merged**